tensorflow_addons/optimizers/weight_decay_optimizers.py

```diff
@@ -114,9 +114,7 @@ def minimize(self,
                 to all variables in var_list.
             name: Optional name for the returned operation.
         Returns:
-            An Operation that updates the variables in `var_list`.  If
-            `global_step` was not `None`, that operation also increments
-            `global_step`.
+            An Operation that updates the variables in `var_list`.
```

Comment:
do you mind following the doc in core TF?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/optimizer_v2/optimizer_v2.py#L307-L309

Reply:
@WindQAQ 

Thanks! I got it.
But I have a question about the document `The iterations will be automatically increased by 1` .
There is no `iterations` variable in `DecoupledWeightDecayExtension` , so I'm not sure what is the means of this description.

Reply:
Ohoh, i see. Never mind.